### ISSUE TITLE
Exercise2-Answer.ipynb: it should get acc instead of accuracy

### ISSUE THREAD

**Issue by yohei1126** [lmoroney/dlaicourse#96]
```
class myCallback(tf.keras.callbacks.Callback):
  def on_epoch_end(self, epoch, logs={}):
    if(logs.get('accuracy')>0.6):
      print("\nReached 60% accuracy so cancelling training!")
      self.model.stop_training = True
```

This does not work. `logs.get('accuracy')` always returns `None`.
It should be

```
class myCallback(tf.keras.callbacks.Callback):
  def on_epoch_end(self, epoch, logs={}):
    if(logs.get('acc')>0.6):
      print("\nReached 60% accuracy so cancelling training!")
      self.model.stop_training = True
```

**Comment by p14jeffwest** [comment]
Yes, I had a sames result.
logs.get('accuracy') made error but logs.get('acc') is ok.

**Comment by geetanjaligg** [comment]
This is because of Tensorflow 1. If you upgrade it to Tensorflow 2, 'accuracy' should work.
